design/20221205-memory-management.md

```diff
@@ -321,37 +321,37 @@ Test the memory spike caused by the initial LIST-ing of `Secret`s, the size of c
 
 Create 300 cert-manager unrelated `Secret`s of size ~1Mb:
 
-![alt text](/design/images/20221205-memory-management/createsecrets.png)
+![screenshot of a terminal using dd to create generic kubernetes secrets `foo-{0..300}`](/design/images/20221205-memory-management/createsecrets.png)
```

Comment:
alt text should be helpful to blind users or users with images off (I use a screen reader at times and often browse the web w/ images off)